pkg/machine/qemu/machine.go

```diff
@@ -372,3 +373,51 @@ func (v *MachineVM) SSH(name string, opts machine.SSHOptions) error {
 
 	return cmd.Run()
 }
+
+// List lists all vm's that use qemu virtualization
+func List() ([]*machine.ListResponse, error) {
```

Comment:
List() should take a `opts ListOptions` even if there are none today.  It allows for future growth without breaking api.